### ISSUE TITLE
Broken info.xml

### ISSUE THREAD

**Issue by demom** [nextcloud/survey_client#37]
Hi,

appinfo/info.xml is broken in current git, contains double namespace tags which breaks a trim in lib/private/AppFramework/App.php row 64.
### error from lib/private/AppFramework/App.php

{"reqId":"cbE7SnxwefNH4EPCvjJo","remoteAddr":"185.18.84.139","app":"PHP","message":"trim() expects parameter 1 to be string, array given at \/var\/www\/nextcloud\/lib\/private\/AppFramework\/App.php#64","level":3,"time":"2016-10-24T12:29:07+00:00","method":"PROPFIND","url":"\/cloud\/remote.php\/webdav\/","user":"--","version":"9.2.0.4"}
### appinfo/info.xml double namespaces

```
<namespace>Survey_Client</namespace>
<licence>AGPL</licence>
<author>Joas Schilling, Björn Schiessle</author>
<version>0.1.5</version>
<namespace></namespace>
```

**Comment by nickvergessen** [comment]
Yeah I experienced the same lately.
Do you want to make a pull request to fix this?

**Comment by demom** [comment]
Sure, right away

**Comment by MorrisJobke** [comment]
@schiessle @nickvergessen

**Comment by MorrisJobke** [comment]
Ah - fixed with #39
